autobotAI_integrations/integrations/rapid7/python_sdk_clients.yml

```
@@ -2,3 +2,9 @@
   import_library_names:
     - autobotAI_integrations.integrations.rapid7
   pip_package_names: []
+
+- name: rapid7_console
+  import_library_names:
+    - autobotAI_integrations.integrations.rapid7
+  pip_package_names: []
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)